### PR TITLE
iptables rule to forward netmaker traffic

### PR DESCRIPTION
**Pull request by mattkasun**
## Describe your changes
counteracts docker setting default forwarding policy to deny nftables not implemented

## Provide link to Netmaker PR if required
N/A
## Provide testing steps
see GRA-1535

## Checklist before requesting a review
- [x] My changes affect only 10 files or less.
- [x] I have performed a self-review of my code and tested it.
- [ ] If it is a new feature, I have added thorough tests, my code is <= 1450 lines.
- [ ] If it is a bugfix, my code is <= 200 lines.
- [x] My functions are <= 80 lines.
- [ ] I have had my code reviewed by a peer.
- [x] My unit tests pass locally.
- [ ] Netclient & Netmaker are awesome.

### CODE REVIEW

File: nmproxy/router/nftables_linux.go [hunk 1]
```diff
@@ -228,6 +228,10 @@ func (n *nftablesManager) CreateChains() error {
 	return nil
 }
 
+func (n *nftablesManager) ForwardRule() error {
+	return errors.New("not implmented")
+}
```
Comment (abhishek9686) [inline nmproxy/router/nftables_linux.go:233]: typo`implemented`

File: nmproxy/router/firewall.go [hunk 2]
```diff
@@ -73,3 +75,14 @@ func Init() (func(), error) {
 	}
 	return fwCrtl.FlushAll, nil
 }
+
+func EnableForwardRule() error {
```
Comment (abhishek9686) [inline nmproxy/router/firewall.go:79]: func comment missing

File: nmproxy/router/firewall.go [hunk 3]
```diff
@@ -34,6 +34,8 @@ const (
 type firewallController interface {
 	// CreateChains  creates a firewall chains and jump rules
 	CreateChains() error
+	// InsertForwardingRule inserts forwarding rules
+	ForwardRule() error
```
Comment (abhishek9686) [inline nmproxy/router/firewall.go:38]: comment needs to be changed

**Review by abhishek9686 — CHANGES_REQUESTED** [review]

**Review by abhishek9686 — APPROVED** [review]

**Review by theguy951357 — COMMENTED** [review]
critical list passes, but
docker netclient does not have connectivity.

**Pull request merged**